content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -379,32 +379,82 @@ Introspection does not currently support the [MongoDB connector](../../../concep
+Formats the Prisma schema file, which includes validating, formatting, and persisting the schema.
 
-<Admonition type="warning">
+#### Arguments
 
-**Warning**: The command will overwrite the current `schema.prisma` file with the new schema. Any manual changes or customization will be lost. Be sure to back up your current `schema.prisma` file before running `introspect` if it contains important modifications.
+| Argument   | Required | Description                                                                                                                                         | Default                                     |
+| ---------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------- |
+| `--schema` | No       | Specifies the path to the desired `schema.prisma` file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`, `./prisma/schema.prisma` |
 
-</Admonition>
+#### Examples
 
-#### Prerequisites
+##### Validate a schema without errors
 
-Before using the `introspect` command, you must define a valid [`datasource`](../../concepts/components/prisma-schema/data-sources) within your `schema.prisma` file.
+<CodeWithResult>
+<cmd>
 
-For example, the following `datasource` defines a SQLite database file within the current directory:
+```terminal
+npx prisma format
+```
 
-```prisma
-datasource db {
-  provider = "sqlite"
-  url      = "file:my-database.db"
-}
+</cmd>
+<cmdResult>
+
+```
+Environment variables loaded from prisma/.env
+Prisma schema loaded from prisma/schema.prisma
+Formatted prisma/schema.prisma in 116ms �
+```
+
+</cmdResult>
+</CodeWithResult>
+
+##### Formatting a schema with validation errors
+
+<CodeWithResult>
+<cmd>
+
+```terminal
+npx prisma format
 ```
 
+</cmd>
+<cmdResult>
+
+```
+Environment variables loaded from prisma/.env
+Prisma schema loaded from prisma/schema.prisma
+Error: Schema parsing
+error: The preview feature "namedConstraints" is not known. Expected one of: filterJson
+  -->  schema.prisma:3
+   |
+ 2 |   provider        = "prisma-client-js"
+ 3 |   previewFeatures = ["filterJson"]
+   |
+
+Validation Error Count: 1
+```
+
+</cmdResult>
+</CodeWithResult>
+
+## <inlinecode>db</inlinecode>
+
+The `db [command]` command connects to your database and adds Prisma models to your Prisma schema that reflect the current database schema.
+
+<Admonition type="warning">
+
+**Warning**: The command will overwrite the current `schema.prisma` file with the new schema. Any manual changes or customization will be lost. Be sure to back up your current `schema.prisma` file before running `db pull` if it contains important modifications.
+
+</Admonition>
```

Comment:
This is wrong. There is not `db` command by itself, only `db pull`. @molebox

Reply:
Hmm, thats what i meant by `db [command]` but i can remove if it doesnt make sense

Reply:
There is no `db [command]` thing - that was not changed at all.